carapax/src/command.rs

```diff
@@ -0,0 +1,254 @@
+use crate::{
+    convert::TryFromUpdate,
+    handler::{Handler, HandlerResult},
+};
+use async_trait::async_trait;
+use shellwords::MismatchedQuotes;
+use std::{collections::HashMap, error::Error, fmt, string::FromUtf16Error};
+use tgbot::types::{Message, Update};
+
+/// A simple commands handler
+#[derive(Default)]
+pub struct CommandDispatcher<C> {
+    handlers: HashMap<String, Box<dyn Handler<C, Input = Command, Output = HandlerResult> + Send>>,
```

Comment:
```rust
pub(crate) type = Box<dyn Handler<C, Input = Command, Output = HandlerResult> + Send>
```
```suggestion
    handlers: HashMap<String, BoxedHandler>,
```
We can see this type in other code, can't we?